src/H5TSdevelop.h

```diff
@@ -11,14 +11,15 @@
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
 /*
- * This file contains public declarations for the H5TS (threadsafety) module.
+ * This file contains public declarations for the H5TS (threadsafety) developer
+ *      support routines.
  */
 
-#ifndef H5TSpublic_H
-#define H5TSpublic_H
+#ifndef H5TSdevelop_H
+#define H5TSdevelop_H
 
-/* Public headers needed by this file */
-#include "H5public.h" /* Generic Functions                    */
+/* Include package's public header */
+/* #include "H5TSpublic.h" */
```

Comment:
Any need to keep this line around?

Reply:
There are similar placeholders in the library, so that's why I had left it in.   But, there's plenty of examples of how / where to include a public header file for an interface, so I'll take it out.